kytos/core/controller.py

```diff
@@ -151,16 +151,15 @@ def create_pidfile(self):
                 # If kill() doesn't return an error, there's a process running
                 # with the same PID. We assume it is Kytos and quit.
                 # Otherwise, overwrite the file and proceed.
-                self.log.error("PID file %s exists. Delete it if Kytos is not"
-                               "running. Aborting.")
-                sys.exit(os.EX_CANTCREAT)
+                error_msg = ("PID file {} exists. Delete it if Kytos is not "
```

Comment:
This message is logged too, when the SystemExit is raised the error message is given and the controller will display this in the log.